services/verify.js

```diff
@@ -0,0 +1,67 @@
+const {FactomCli} = require('factom');
+const crypto = require('crypto');
```

Comment:
Why are we importing this package when the factom package also has utils for creating hashes?

Reply:
I don't see any reference to sha256 from going over the README. Do you have an example available?

Reply:
`import factomUtils from "factom/src/util";`

Reply:
```javascript
const hashFile = (file) => {
  return factomUtils
    .sha256(file);
};
```

Reply:
FactomUtils is not exported, so this might break: https://github.com/PaulBernier/factomjs/blob/master/src/factom.js